build_tools/benchmarks/run_benchmarks_on_android.py

```diff
@@ -91,6 +91,50 @@ def get_git_commit_hash(commit: str) -> str:
                                         os.path.realpath(__file__)))
 
 
+def get_android_device_model(verbose: bool = False) -> str:
```

Comment:
We should move these functions to a new file under `common/`, like `common/android.py` or something.

Reply:
Put them into the `common/android_utils.py`.